lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -438,16 +377,14 @@ public function render_metrics() {
     <?php
   }
 
-  public function render_verified_email() {
-    $v = absint( $this->options->get( 'requires_verified_email' ) );
-
-    echo $this->render_a0_switch( "wpa0_verified_email", "requires_verified_email", 1, 1 == $v );
-?>
-      <div class="subelement">
-        <span class="description"><?php echo __( 'Mark this if you require the user to have a verified email to login', 'wp-auth0' ); ?></span>
-      </div>
-    <?php
+  public function render_verified_email( $args ) {
+    $this->render_switch( $args[ 'label_for' ], $args[ 'opt_name' ] );
+    $this->render_field_description(
+      __( 'Require new users to verify their email before logging in. ', 'wp-auth0' ) .
+      __( 'This will disallow logins from social connections that do not provide email (like Twitter)', 'wp-auth0' )
```

Comment:
Are users even created when this case happens? i.e. I sign up using Twitter. I won't be able to log in ever??

Reply:
Users are created if an email is present in their userinfo. If this is on and you sign in with Twitter, you're rejected, correct.

Reply:
@joshcanhelp But is the "wordpress user" created in that case?

Reply:
No. No user created, no login allowed